src/androidTest/java/com/b44t/messenger/uitests/offline/SharingTest.java

```diff
@@ -93,7 +93,7 @@ public void testShareFromScreenshot() {
       }
     }
     Uri uri = Uri.parse("content://" + BuildConfig.APPLICATION_ID + ".attachments/" + Uri.encode(pngImage));
-    DcHelper.sharedFiles.put("/" + pngImage, 1);
+    DcHelper.sharedFiles.put(pngImage, 1);
```

Comment:
For explanation: Since `AttachmentsContentProvider.openFile()` (which checks `sharedFiles`) now uses `getPathSegments().get(0);` instead of `getPath();`, the leading `/` won't be included there. So, it's easier for  `AttachmentsContentProvider.openFile()` to have it without the leading `/`.